src/Proto.Actor/LocalContext.cs

```diff
@@ -124,18 +134,26 @@ public PID SpawnNamed(Props props, string name)
 
         public void SetBehavior(Receive receive)
         {
-            _behavior.Clear();            
+            _behavior = null;

```

Comment:
Not sure if clear is better than setting it to null.  I guess I should just clear it if it's already made.